ieee.csl

```diff
@@ -177,7 +177,7 @@
       <if type="paper-conference speech" match="any">
         <choose>
           <!-- Published Conference Paper -->
-          <if variable="collection-editor editor editorial-director issue page volume" match="any">
```

Comment:
@bwiernik made this change explicitly in https://github.com/citation-style-language/styles/pull/4867 -- I just want to confirm that this is no longer necessary now that we have event-title separately from container-title, yes?